CODE_OF_CONDUCT.md

```diff
@@ -1,40 +0,0 @@
-# Welcome to Fox's Magisk Module Manager community
-
-These guidelines set forth our expectations for our communities, not limiting but including our GitHub Repository, our Telegram groups, and other affiliated groups/communities. They really boil down to three principles:
-
-1) Don't be a dick.
-2) A dick do not be.
-3) Review rules 1 and 2.
-
-We believe in free speech, and freedom of expression, insofar that it doesn't interfere with others' right to the same. Contributors, community members, developers, and users alike are welcome here, but you must extend the same welcome to other members. So long as you can follow those three principles, you are welcome here, whether it's to hand out, lurk, or more directly contribute.
-
-## What we do not allow
-
-Absolutely no harrassment including, but not limited to, harassment based on the basis of sexuality, gender, race, or other beliefs shall be tolerated. We want everyone to feel safe and welcome here, and when some community members harass others, that is made much harder.
-
-Harassment is defined as, as per the english dictionary:
-```
-noun: harassment; plural noun: harassments
-  aggressive pressure or intimidation.
-  "they face daily harassment and assault on the streets"
-```
-
-We will also not tolerate any illegal activity, including but not limited to, copyright infringement, license infringement, threats, violence, doxxing, or any activity not legal in any team member's or developers home country.
-
-## What we do allow
-
-We believe free speech and discussions are vital to the health of a community. When everyone can speak freely, we all get along much better. Keep in mind, you must extend that same freedom to everyone else. That means to allow others to speak their mind just as you have, without attempting to stifle their same freedom of speech or expression.
-
-### Specifics for code contributions
-
-All code submitted must follow existing code standards, including but not limited to, formatting, licensing, and code style. Code must also be the submitters own work or used under license, and the submitter must make a good faith effort to test and ensure it works correctly.
-
-## Enforcement
-
-While the police will not show up at your house for violating our rules, we reserve the right but not the obligation to issue warnings and/or ban users as we see fit for violations of this Code of Conduct.
```

Comment:
These aren't rules governed here, but rather a duplication of laws police will arrest you for in any and all jurisdictions.
For the police to not engage in the ruling of community rules is otherwise a given, and not something that needs mention.
Anyone can be banned for any reason at any time anyhow.